.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+######Fixes #[ISSUE]
+
+####Changes proposed:
+* [ ] Add
+* [ ] Prepare
```

Comment:
I thought that even half commits are usual, like this one. It still needs improvment(or help), but the others already have something they can use as a base.